phplib/AdminStringUtil.php

```diff
@@ -88,7 +88,7 @@ static function internalize($text, $preserveAccent) {
 
   // Generic purpose cleanup of a string. This should be true of all columns of all tables.
   static function cleanup($s) {
-    $s = trim($s);    
+    $s = trim($s);
```

Comment:
Editorul meu elimină spațiile de la sfârșit de rând când salvează un fișier și nu am observat modificările astea înainte de commit. Oricum, sunt puține și nu cred că afectează cu nimic.

Reply:
Da, doar că de obicei prefer să nu pun mai multe tipuri de schimbări într-un singur commit, poate deveni greu de urmărit diff-ul (dar aici sunt puține modificări).

Reply:
Principiile sunt bune, când nu te împiedică de la treabă. :-) Nu e nici un fel de problemă